src/__tests__/index.test.js

```diff
@@ -0,0 +1,22 @@
+import * as functions from '../index';
```

Comment:
Added this, thinking it might be useful.

Was thinking we could also assert on the function signatures/return types, but I guess Flow does that for you?

Reply:
I'm not sure how you would even assert the function signatures other than rewriting the unit tests for them. I don't think we need these tests at all

Reply:
No worries, will remove.

Reply:
Will merge this one now and remove in the next PR which I'm pushing now